src/Adapter/Product/AdminProductDataProvider.php

```diff
@@ -286,12 +286,15 @@ public function getCatalogProductList($offset, $limit, $orderBy, $sortOrder, $po
             $sqlOrder = array('id_product ASC');
         }
 
+        $sqlGroupBy = [];
```

Comment:
Currently, we initialize arrays with the long syntax form. `[]` should be `array()`